pipeline_steps/common.groovy

```diff
@@ -1629,6 +1629,10 @@ void runReleasesPullRequestWorkflow(String baseBranch, String prBranch, String j
 
   String description = "Release tests passed, merging..."
   gate.updateStatusAndMerge(description, statusContext)
+
+  if (prType == "release"){
+    updateDependents(REPO_NAME)
```

Comment:
Failures here shouldn't cause the release build to fail so I'd suggest wrapping this in exception handling that notifies RE if there are problems.

I'd suggest moving this to the line below `createRelease(componentText, has_rc_branch)` so that it is with all related code. Doing so would also move it within the `dir` block which is required to ensure the call to `component` on line 1932 works.

Reply:
I might be missing something...  if I were to put the call to `updateDependents` in the dir(REPO_NAME) block, it would no longer have the "releases" dir at "./". and it wouldn't work anymore.  But to your point, the releases dir is not verified anywhere, including when `getComponentChange` is called prior to `updateDependents`

Reply:
Ignore this, I'm talking nonsense. I was having a `1 + 1 = 3` moment.